src/Microsoft.Azure.ServiceBus/ExceptionReceivedEventArgsAction.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.Azure.ServiceBus
+{
+    static class ExceptionReceivedEventArgsAction
```

Comment:
I'm sure it doesn't fit into the mainline scenario. But even for a very simple use case wherein a customer wants to keep a count of exceptions occurred during each operation, he would need to segregate all the exceptions received based on this action.